### ISSUE TITLE
virtial: User is getting prompted to enter value for a non existing field in "Volunteer" section

### ISSUE THREAD

**Issue by nss-popuri** [HTBox/crisischeckin#546]
Steps to follow:
a) Login to the Crisis Checkin site
b) Select a value from the "Disaster" dropdown in "My Organization Resource" section
c) DO NOT enter/select any other values
d) Click on "Checkin" button in "My Organization Resource" section
e) Observe the message displayed in "Make a new Commitment" section. Attached word file is having the required screen shots
[Error_ScreenShot.docx](https://github.com/HTBox/crisischeckin/files/249981/Error_ScreenShot.docx)

Actual Result: "The Resource Type field is required" is displayed in "Make a new Commitment" section
Expected Result: This message should not be displayed as "Resource Type" is a field present in My Organization Resource" section But not in "Make a new Commitment" section

**Comment by NaveenP94** [comment]
You are correct the error messages for the Make new commitment also appear under report new data and vice versa, those two form entities need to be decoupled.
